pkg/skaffold/sync/sync.go

```diff
@@ -328,6 +328,10 @@ func Perform(ctx context.Context, image string, files syncMap, cmdFn func(contex
 		return nil
 	}
 
+	if len(namespaces) == 0 {
```

Comment:
is there any way to add a test for this? not sure if that's possible with the current unit test setup

Reply:
done